theme/themes/eea/extras/header.less

```diff
@@ -298,27 +298,33 @@
 
 /* Subsite */
 .eea.header .logo-wrapper {
+  position: relative;
   display: flex;
   align-items: center;
-  gap: 2em;
+  margin-top: @mobileLogoMarginTop;
+  gap: 30px;
+
+  .eea-logo {
+    margin: 0;
+  }
 }
 
 .eea.header .subsite-logo {
-  margin-top: @mobileLogoMarginTop;
-  font-size: @mobileSubsiteFontSize;
+  position: relative;
+  color: @subsiteColor;
   font-family: @subsiteFontFamily;
+  font-size: @mobileSubsiteFontSize;
   font-weight: @subsiteFontWeight;
-  color: @subsiteColor;
-  position: relative;
+  line-height: @subsiteLineHeight;
 
   &:before {
     position: absolute;
+    top: 50%;
+    left: -15px;
```

Comment:
@kreafox  also here these values might need to be changed in another thematic website